src/scripts/placeDE-overlay.user.js

```diff
@@ -10,19 +10,17 @@
 // @downloadURL  https://github.com/PlaceDE-Official/place-overlay/raw/main/src/scripts/placeDE-overlay.user.js
 // ==/UserScript==
 
-var overlayImage = null;
-if (window.top !== window.self) {
-    window.addEventListener('load', () => {
-        const canvasContainer = document.querySelector("garlic-bread-embed").shadowRoot.querySelector("div.layout").querySelector("garlic-bread-canvas").shadowRoot.querySelector("div.container");
+let overlayImage = null;
+if(window.top !== window.self) {
+    window.addEventListener("load", function() {
+        const canvasContainer = document.querySelector("garlic-bread-embed").shadowRoot.querySelector("garlic-bread-canvas").shadowRoot.querySelector(".container");
```

Comment:
Great shortening!

Reply:
sorry, is just some kink in my programming

Reply:
changed back to smaller size